src/Main.elm

```diff
@@ -59,8 +90,8 @@ getBombCount difficulty =
         Expert ->
             99
 
-        Custom _ _ count ->
-            count
+        Custom { bombs } ->
+            bombs
```

Comment:
i got the feeling you didn't like these `getBombCount` and `getDimensions` functions and favored creating values like `beginnerSettings` rather than relying on the `Beginner` type. maybe this kind of thing seems backwards, if you're coming from a more object-oriented perspective, but in FP, you tend to creating functions like this, mapping one type to another. maybe you'll see the benefits in the code down below.